ACE/ace/CDR_Base.inl

```diff
@@ -145,7 +145,11 @@ ACE_CDR::swap_8 (const char* orig, char* target)
     && !defined(ACE_LACKS_INLINE_ASSEMBLY)
   ACE_REGISTER unsigned long x =
     * reinterpret_cast<const unsigned long*> (orig);
+#if defined (__MINGW64__)
```

Comment:
This code is not compiled on native MinGW due to...
ace/config-win32-mingw64.h:82:#   define ACE_LACKS_INLINE_ASSEMBLY
...if that's not active for the cross-compiler, something may need to be corrected with the configuration.